### PR TITLE
[Merged by Bors] - chore: deprime `induction` and `cases` up to `Algebra.Lie`

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/BigOperators/Group/Multiset.lean [hunk 1]
```diff
@@ -106,18 +106,16 @@ theorem prod_replicate (n : ℕ) (a : α) : (replicate n a).prod = a ^ n := by
 @[to_additive]
 theorem prod_map_eq_pow_single [DecidableEq ι] (i : ι)
     (hf : ∀ i' ≠ i, i' ∈ m → f i' = 1) : (m.map f).prod = f i ^ m.count i := by
-  induction' m using Quotient.inductionOn with l
-  simp [List.prod_map_eq_pow_single i f hf]
+  induction m using Quotient.inductionOn; simp [List.prod_map_eq_pow_single i f hf]
```
Comment (semorrison) [inline Mathlib/Algebra/BigOperators/Group/Multiset.lean:109]:
```suggestion
  induction m using Quotient.inductionOn
  simp [List.prod_map_eq_pow_single i f hf]
```
and below

File: Mathlib/Algebra/DirectLimit.lean [hunk 2]
```diff
@@ -170,9 +170,8 @@ lemma lift_injective [IsDirected ι (· ≤ ·)]
   · apply Function.injective_of_subsingleton
   simp_rw [injective_iff_map_eq_zero] at injective ⊢
   intros z hz
-  induction' z using DirectLimit.induction_on with _ g
-  rw [lift_of] at hz
-  rw [injective _ g hz, _root_.map_zero]
+  induction z using DirectLimit.induction_on with
+  | ih _ g => rw [lift_of] at hz; rw [injective _ g hz, _root_.map_zero]
```
Comment (semorrison) [inline Mathlib/Algebra/DirectLimit.lean:174]: Please don't use `;` unless you're getting the entire proof on one line, I think.

**Comment by semorrison** [comment]
bors d+

**Comment by Parcly-Taxel** [comment]
bors merge

**Comment by Parcly-Taxel** [comment]
bors merge

**Comment by Parcly-Taxel** [comment]
bors merge

**Comment by Parcly-Taxel** [comment]
bors merge

**Comment by bryangingechen** [comment]
Sorry, the change to Mathlib.lean you made in [65c269d](https://github.com/leanprover-community/mathlib4/pull/16189/commits/65c269d74dfbce1b5b5553b74a9fb6240de20710) is most likely a response to a build failure on `master` itself caused by an accidental force push which [was just now reverted](https://github.com/leanprover-community/mathlib4/pull/16242/). If you undo the change in that commit and then merge `master` now things should hopefully work out.

**Comment by sgouezel** [comment]
Can you merge master, fix the build, and then send it back to bors?
bors r-
bors d+

**Comment by Parcly-Taxel** [comment]
bors merge

**Pull request closed without merging**